web_demo/js/pitch/PitchUpdater.js

```diff
@@ -0,0 +1,49 @@
+var PitchUpdater = function(audioContext) {
+    /**
+     * pitchData 객체를 만들어 반환한다. 현재는 콘솔에 출력까지 한다.
+     * @returns {{isConfident: boolean, pitch: number, note: string}}
```

Comment:
comment with returns annotation should describe about the value itself not only type.